File: MealPlanApp/MealPlanApp.Domain/Models/MealPlan.cs
namespace MealPlanApp.Domain.Models
{
    public class MealPlan
    {
        public Guid Id { get; set; }
        public string PatientName { get; set; } = null!;
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int TotalCalories { get; set; }
        public string? Notes { get; set; }
        public ICollection<Meal> Meals { get; set; } = new List<Meal>();
    }
}

File: MealPlanAPP-Front/mealplans-web/src/types/MealPlan.ts
export interface Meal {
  id?: string
  name: string
  time: string
  description?: string
}

export interface MealPlan {
  id?: string
  patientName: string
  startDate: string
  endDate: string
  totalCalories: number
  notes?: string
  meals: Meal[]
}